examples/my-pod.yaml

```diff
@@ -7,18 +7,20 @@ metadata:
     k8s.v1.cni.cncf.io/networks: default-sriov-net
 spec:
   nodeSelector:
-    kubernetes.io/hostname: worker-237
+    kubernetes.io/hostname: worker-229
   containers:
   - name: appcntr1
-    image: registry.access.redhat.com/ubi9/ubi:latest
+    image: ghcr.io/ovn-kubernetes/kubernetes-traffic-flow-tests:latest
     command: ['/bin/sh', '-c', 'sleep infinity']
     imagePullPolicy: Always
     securityContext:
-      privileged: true
+      priviledged: true
```

Comment:
this was just changed by 13b0558d64934eeabbf7488ce937376147adb4ca

Are you sure you get the right spelling here?